src/main/kotlin/model/BoardMaker.kt

```diff
@@ -0,0 +1,15 @@
+package model
+
+class BoardMaker {
+    fun run(boardHeight: BoardHeight, boardWidth: BoardWidth, mine: Mine): List<String> {
+        val board = MutableList(boardHeight.height * boardWidth.width - mine.size) { TILE }
+        val mines = MutableList(mine.size) { MINE }
+        board.addAll(mines)
+        return board.shuffled()
+    }
```

Comment:
많은 코틀린의 타입은 불변이 기본값이고 List도 그중 하나입니다. 이런 불변 타입을 사용하는 것의 장점을 온전히 누리기 위해 전 코틀린에서는 가능하면 var이나 Mutable 타입 사용을 지양하려고 하고 있습니다. 🙂

위 경우도 Mutable한 List가 아니도록 구현할 수 있을 것 같아 다음과 같이 작성하는 것은 어떨지 제안드려봐요!

```suggestion
    fun run(boardHeight: BoardHeight, boardWidth: BoardWidth, mine: Mine): List<String> {
        val board = List(boardHeight.height * boardWidth.width - mine.size) { TILE }
        val mines = List(mine.size) { MINE }
        return (board + mines).shuffled()
    }
```

Reply:
그냥 +로 쉽게 합쳐지는군요 ^^;